lucid/misc/io/saving.py

```diff
@@ -320,6 +334,9 @@ def save(thing, url_or_handle, allow_unsafe_formats=False, save_context: Optiona
     if result is not None and "url" in result and result["url"].startswith("gs://"):
         result["serve"] = "https://storage.googleapis.com/{}".format(result["url"][5:])
 
+    if isazure(result["url"]):
+        result["serve"] = url
```

Comment:
ha, this is good to know

Reply:
occurred to me this was a bit of a tautology, but i left it in there in case there were any scripts making use of "serve"